prow/github/client.go

```diff
@@ -663,3 +664,27 @@ func (c *Client) FindIssues(query string) ([]Issue, error) {
 	}, &issSearchResult)
 	return issSearchResult.Issues, err
 }
+
+// GetContents uses github repo contents API to retrieve the content of a file.
+// Todo: Support retrieve a directory
+func (c *Client) GetFile(org, repo, filepath string) (string, error) {
+	c.log("GetRemoteFile", org, repo, filepath)
+	var res struct {
+		Object map[string]string `json:"object"`
```

Comment:
Instead of `map[string]string` explicitly define the pieces you expect to receive. Then you can remove the check below.